pages/[network]/create-bounty.tsx

```diff
@@ -305,7 +375,20 @@ export default function PageCreateIssue() {
                     }
                   />
                 </div>
-                <div className="col"></div>
+                
+                <div className="col-6 mt-n2">
+                  <TokensDropdown 
+                    defaultToken={BEPRO_TOKEN} 
+                    tokens={customTokens} 
+                    canAddToken={
+                      activeNetwork?.networkAddress === CONTRACT_ADDRESS ? 
+                      ALLOW_CUSTOM_TOKENS :
```

Comment:
_i think_ that with V2 we can have all networks be a db value -- but this can be a change added later